### DISCUSSION TITLE
WARNING: it is strongly recommended to first generate an in silico-predicted library in a separate pipeline step and then use it to process the raw data, now without activating FASTA digest

### DISCUSSION THREAD

**Discussion by jeewan-rijal** [vdemichev/DiaNN#1239] [Q&A] [answered]
Hello, thanks a lot for this excellent tool and regular updates.
In the latest versions of DIA-NN, we see this warning as shown in the title.
Does it apply to all datasets? Do we now need to run samples in 2 steps?
Or can we go from the fasta to the final result as we did before?
What difference should we expect?

**Comment by vdemichev** [comment, marked as answer]
Hi,

Yes, please always heed DIA-NN's warnings and see if they make sense. If it says 'strongly recommended', then please definitely address the warning. I have just posted also an explanation for the reasons for this particular warning: https://github.com/vdemichev/DiaNN/issues/1238. That being said, in 1.9.2 'on-the-fly' one-step approach should work fine.

Best,
Vadim
